config/default/manager_auth_proxy_patch.yaml

```diff
@@ -11,6 +11,12 @@ spec:
       containers:
       - name: kube-rbac-proxy
         image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.7.0
+        securityContext:
+          allowPrivilegeEscalation: false
+          runAsNonRoot: true
```

Comment:
btw, the `runAsNonRoot` is already defined at the pod level
https://github.com/codeready-toolchain/member-operator/blob/ec800b33605d265754c7791b6345cc229f62a27e/config/manager/manager.yaml#L27-L28
I guess that we don't have to re-define it also for containers separately

Reply:
from the documentation, "Security settings that you specify for a Container apply only to the individual Container, and they override settings made at the Pod level when there is overlap." I interpreted that we need to re-define it at container level again to avoid vulnerability.

Reply:
> The security settings that you specify for a Pod apply to all Containers in the Pod

However if an individual container template also specifies the security context then that one from the container template will override the one specified for the pod.

But this specific `config/default/manager_auth_proxy_patch.yaml` template doesn't have either the pod or the individual container security contexts specified. I guess this is why snyk complains about it.

Reply:
tbh, we could easily drop the patch and move the definition of  the auth proxy directly to the manifest of the Deployment - there is no benefit in having it separate. We just reused whatever was generated by operator-sdk. This could also make the management easier

Reply:
done in [5e08ac7](https://github.com/codeready-toolchain/member-operator/pull/465/commits/5e08ac7f00d983ba2633997088a5f06111ae7c5c)